src/js/view/view.js

```diff
@@ -905,6 +946,22 @@ function View(_api, _model) {
 
                 addClass(_playerElement, 'jw-flag-floating');
 
+                if (mobileFloatIntoPlace) {
+                    // Creates a dynamic animation where the top of the current player
+                    // Smoothly transitions into the expected floating space in the event
+                    // we can't start floating at 62px
+                    style(_wrapperElement, {
+                        transform: `translateY(-${FLOATING_TOP_OFFSET - playerBounds.top}px)`
+                    });
+
+                    setTimeout(() => {
```

Comment:
i know this many style calls is usually a no-no but given we are only `transform`ing the element, this will only impact `composite` step of the browser rendering. it's by far the cheapest as there is no heavy lifting, simply a movement of the painted layer